code/modules/spells/targeted/projectile/fireball.dm

```diff
@@ -25,12 +25,50 @@
 	var/ex_light = 2
 	var/ex_flash = 5
 
+	level_max = list(Sp_TOTAL = 7, Sp_SPEED = 4, Sp_POWER = 1)
+
+	var/list/explosion_by_power = list( //Explosion stats corresponding to the spell's power level. Level 1 explosion is the first list (-1,1,2), level 2 is the second, ...
+		list(-1, 1, 2, 5)
+	)
```

Comment:
Yep, originally I made the explosion bigger with every power upgrade but then I decided to do something else